tests/cms/views/view_config.py

```diff
@@ -814,7 +814,7 @@
                         "form-0-language": 1,
                         "form-1-title": "",
                         "form-1-text": "",
-                        "form-1-language": 2,
+                        "form-1-language": 6,
```

Comment:
@JoeyStk Regarding 4:
I wondered why the tests of the push notifications failed after I added a new language tree node - in theory, I didn't change anything related to push notifications.
Then, I noticed that the push notification formset expects the forms to be in the same order as the language tree... and since I added the new node Easy German (with id 6) in the language tree right after German (where previously the second language was English), the formset was confused because in the second form now there was language with id 2 but the second language in the tree is now de-si with id 6... so I just changed the test config to adapt to this new language tree structure...
Not sure whether I explained that well, to me it's still kind of confusing nevertheless :see_no_evil:

Reply:
Aha! That's very good to know. I never would have guessed this :thinking: